src/cloud/release-notes/ece-release-notes.md

```diff
@@ -17,10 +17,39 @@ The `{{site.data.var.ct}}` package uses the following release versioning sequenc
 {:.bs-callout-info}
 See [Upgrades and patches]({{ site.baseurl }}/cloud/project/project-upgrade-parent.html) for information about updating to the latest release of the `{{site.data.var.ct}}` package.
 
-## v2002.1.2
+## v2002.1.3
 *Release date: {{ site.data.var.ece-release-date }}*<br/>
 
-<!--Add release notes below-->
+**Infrastructure updates**–
+
+-  {:.new}Added support for Elasticsearch 7.9 and Redis 6 for compatibility with upcoming Magento releases.<!--MCLOUD-7191-->
+
+-  {:.new}Added ece-tools support for the read-only `pub/static` directory when static content is set to deploy in the build stage.<!--MC-37699-->
+
+-  {:.fix}Updated the ece-tools `composer.json` to add a required dependency for the Magento Quality Patches package. This fixes a circular dependency that existed between the ece-tools and magento-cloud-patches packages.<!--MCLOUD-6910-->
+
+**Validation and log improvements**–
+
+-  {:.new}Added search-engine validation to ensure that `elasticsearch` is set for Magento Commerce Cloud 2.4 and later. If the validation fails, the deployment is stopped with an error message suggesting fixes for the issue.<!--MCLOUD-6937-->
```

Comment:
It's warning not a critical issue. Redeploy wouldn't stop. @NadiyaS May you please confirm

Reply:
it should stop redeployment as Magento 2.4 won't work without ES

Reply:
Yeah. You are right. I was thinking it's another elastic validation.